tools/depends/native/TexturePacker/src/TexturePacker.cpp

```diff
@@ -185,13 +264,65 @@ void TexturePacker::CreateSkeletonHeader(CXBTFWriter& xbtfWriter,
   }
 }
 
+void TexturePacker::SubstitudeFile(CXBTFWriter& xbtfWriter, const std::string& fileName)
```

Comment:
Eh, spelling is hard. I'll correct this and other instances later.

Reply:
We can't all be excellent at speling.